.gitignore

```diff
@@ -1 +1,3 @@
 **/*/res/values/secrets.xml
+.gradle/7.3.3/executionHistory/executionHistory.lock
```

Comment:
Do we need the files below i.e. .gradle/7.3.3/executionHistory/executionHistory.lock  and .idea/workspace.xml. Should we add them to gitignore.